__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require('supertest')
+const db  = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+const express = require('express')
+const app = require('../app')
+
+beforeAll(()=>seed(data))
+afterAll(()=>db.end())
+
+describe('api/topics',()=>{
+    test('GET:200 should return an array of topic objects with the properties slug and description',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.length).toBe(3)
+            body.forEach((row)=>{
```

Comment:
We could name row as 'topic' here as that's what we are checking - naming the variable after what it represents makes the code more readable!